driver-core/src/test/java/com/datastax/driver/core/exceptions/ConnectionExceptionTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.core.exceptions;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNull;
+
+public class ConnectionExceptionTest {
+
+    /**
+     * @jira_ticket JAVA-1139
+     */
+    @Test(groups = "unit")
+    public void getHost_should_return_null_if_address_is_null() {
+        assertNull(new ConnectionException(null, "Test message").getHost());
+    }
+
+    /**
+     * @jira_ticket JAVA-1139
+     */
+    @Test(groups = "unit")
+    public void getMessage_should_return_message_if_address_is_null() {
+        assertEquals("[null] Test message", new ConnectionException(null, "Test message").getMessage());
```

Comment:
We might as well remove `[null] ` altogether in that case, that is have the message `"Test message"`.